src/cata_imgui.cpp

```diff
@@ -436,6 +436,11 @@ void cataimgui::draw_colored_text( std::string const &text,
     ImGui::PushID( text.c_str() );
     int startColorStackCount = GImGui->ColorStack.Size;
     ImGuiID itemId = GImGui->CurrentWindow->IDStack.back();
+
+    if( text.empty() ) {
+        ImGui::NewLine( );
+    }
+
```

Comment:
```suggestion
    if( text.empty() ) {
        ImGui::NewLine( );
        return;
    }

    ImGui::PushID( text.c_str() );
    int startColorStackCount = GImGui->ColorStack.Size;
    ImGuiID itemId = GImGui->CurrentWindow->IDStack.back();

```

Reply:
I guess there is no need to do anything if the string is empty and you can simply return early after drawing a new line?

Reply:
Yea, there really is no need to even push an ID only to do nothing and then pop it again.